x-pack/plugins/triggers_actions_ui/public/common/expression_items/of.scss

```diff
@@ -0,0 +1,3 @@
+.actOf__aggFieldContainer {
+  width: $euiSize * 30;
```

Comment:
Was just thinking about this a bit more. I think we should increase the width but not by this much. Let's try to go with $euiSize x 29 here and just assume that there we'll be some really (really) long names that will get trimmed and the user will have to hover on to see the whole name. That way it doesn't look too weird in the cases where field names are not that long.